icons/molar.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Rabiosp"
+  ],
+  "tags": [
+    "molar",
+    "tooth"
+  ],
+  "categories": [
+    "dentistry",
```

Comment:
These categories do not exist. I think https://lucide.dev/icons/categories#medical is probably the one that fits best here.